tests/test_notify_suppliers_with_incomplete_applications.py

```diff
@@ -0,0 +1,218 @@
+#                 'protectionOfDataAtRest': ['csa_ccm'], 
+#                 'resellingType': 'not_reseller', 
+#                 'scaling': False, 
+#                 'serviceBenefits': ['user friendly', 'company specific'], 
+#                 'serviceCategories': [
+#                     'Archiving, backup and disaster recovery', 
+#                     'Compute and application hosting', 
+#                     'Container service', 
+#                     'Content delivery network', 
+#                     'Data warehousing', 
+#                     'Distributed denial of service attack (DDOS) protection', 
+#                     'Firewall', 
+#                     'NoSQL database', 
+#                     'Other database services', 
+#                     'Relational database',
+#                 ], 
+#                 'serviceConstraints': 'None', 
+#                 'serviceDescription': 'Saas', 
+#                 'serviceFeatures': ['one cost delivery', 'money back guarantee'], 
+#                 'serviceName': 'Saas', 
+#                 'standardsCSASTAR': False, 
+#                 'standardsISO28000': False, 
+#                 'standardsISOIEC27001': False, 
+#                 'standardsPCI': False, 
+#                 'status': 'not-submitted', 
+#                 'supplierId': 92277, 
+#                 'supplierName': 'Govdata Ltd', 
+#                 'supportAvailableToThirdParty': True, 
+#                 'supportLevels': 'one off cost', 
+#                 'systemRequirements': ['N/A', 'N/A'], 
+#                 'updatedAt': '2018-05-10T08:22:31.730794Z', 
+#                 'usageNotifications': False, 
+#                 'webChatSupport': 'no', 
+#                 'webInterface': False
+#             }
+#         ]
+#     },
+# ]
+
+# USERS_TEST_CASES = [
+#     {
+#         'users': []
+#     },
+#     {
+#         'users': [
+#             {
+#                 'active': False,
+#                 'emailAddress': 'abc@example.com'
+#             },
+#             {
+#                 'active': True,
+#                 'emailAddress': 'abc@example.com'
+#             },
+#         ]
+#     },
+# ]
+
+MESSAGE_TEST_CASES = (
+    (
+        {
+            'services': []
+        },
+        [
+            {
+                'supplierId': 0,
+                'applicationCompanyDetailsConfirmed': True,
+                'declaration': {'status': 'submitted', 'primaryContactEmail': 'abc@example.com'}
+            },
+            {
+                'supplierId': 1,
+                'applicationCompanyDetailsConfirmed': None,
+                'declaration': {'status': 'submitted', 'primaryContactEmail': 'abc@example.com'}
+            },
+        ],
+        {
+            'users': [
+                {
+                    'active': False,
+                    'emailAddress': 'abc@example.com'
+                },
+                {
+                    'active': True,
+                    'emailAddress': 'abc@example.com'
```

Comment:
Let's make the active and inactive users have different emails so we can assert it's been sent to the right one 😸